google/auth/transport/_mtls_helper.py

```diff
@@ -405,3 +406,30 @@ def client_cert_callback():
 
     # Then dump the decrypted key bytes
     return crypto.dump_privatekey(crypto.FILETYPE_PEM, pkey)
+
+def check_use_client_cert():
+  """Returns whether the client certificate should to be used for mTLS.
+
+  Returns:
+      str:
+          A boolean indicating if client certificate should be used.
+          The value is "true" or "false" or unset.
+          If unset, the function checks if the GOOGLE_API_CERTIFICATE_CONFIG
+          environment variable is set. If it is set, the function returns
+          "true" if the certificate config file contains "workload" section
+          and "false" otherwise.
+  """
+  use_client_cert = os.getenv("GOOGLE_API_USE_CLIENT_CERTIFICATE")
+  ### Check if the value of GOOGLE_API_USE_CLIENT_CERTIFICATE is unset.
```

Comment:
nit: The two comments in this method that start with ### are unnecessary as the code is self-explanatory. These can be removed to improve code clarity.
If you'd like to keep the comments, I'd recommend changing the prefix to "#" since "###" is not a common practice in this repo.